### PR TITLE
Approved to string

### PR DESCRIPTION
**Pull request by perryr16**
change :approved column in apps table to type :string

### CODE REVIEW

File: app/views/favorites/index.html.erb [hunk 1]
```diff
@@ -27,23 +27,23 @@
...
-          <% end %>
-
-        <% end %>
-      </section>
-    <% end %>
 
 
 
   <% end %><br><br>
+  <%  %>
```
Comment (perryr16) [inline app/views/favorites/index.html.erb:34]: delete this

Comment (muydanny) [reply app/views/favorites/index.html.erb:34]: will do

File: db/migrate/20200514204055_change_approved_to_string.rb [hunk 2]
```diff
@@ -0,0 +1,5 @@
+class ChangeApprovedToString < ActiveRecord::Migration[5.1]
+  def change
+    change_column :apps, :approved, :string
```
Comment (perryr16) [inline db/migrate/20200514204055_change_approved_to_string.rb:3]: beacuse approved will have three states, needs to be a string instead of a boolean

Comment (muydanny) [reply db/migrate/20200514204055_change_approved_to_string.rb:3]: Cool, looks good

**Pull request merged**